lib/matplotlib/ticker.py

```diff
@@ -730,19 +733,23 @@ def format_data_short(self, value):
                 (math.floor(math.log10(abs(value))) + 1 if value else 1)
                 - math.floor(math.log10(delta)))
             fmt = f"%-#.{sig_digits}g"
-        return (
-            self.fix_minus(
-                locale.format_string(fmt, (value,)) if self._useLocale else
-                fmt % value))
+        return self._format_maybe_minus_and_locale(fmt, value)
 
     def format_data(self, value):
         # docstring inherited
-        if self._useLocale:
-            s = locale.format_string('%1.10e', (value,))
+        e = math.floor(math.log10(value))
```

Comment:
Probably needs to be `abs(value)` to fix the docs.

Reply:
thanks